samples/cellular/modem_shell/README.rst

```diff
@@ -819,6 +825,32 @@ You can use the ``heap`` command to print kernel and system heap usage statistic
      free:              160
      allocated:          88
 
+----
+
+GPIO pin pulse counter
+======================
+
+MoSh command: ``gpio_count``
+
+You can use the command to count pulses on a given GPIO pin.
```

Comment:
Please be more specific about how the counting actually happens. I am guessing you are counting rising edges?

Reply:
Good point, I'll add it. Yes, it is counting rising edges.